Lib/pydoc.py

```diff
@@ -1102,7 +1159,7 @@ def docroutine(self, object, name=None, mod=None,
             doc = doc and '<dd><span class="code">%s</span></dd>' % doc
             return '<dl><dt>%s</dt>%s</dl>\n' % (decl, doc)
 
-    def docdata(self, object, name=None, mod=None, cl=None):
+    def docdata(self, object, name=None, mod=None, cl=None, *ignored):
```

Comment:
Not sure why `*ignored` is needed 🤔

Reply:
Because `homecls` is now passed, but it is only used in `docroutine()`. Perhaps `mod` and `cl` could be included in it, as they are not used too. Some other methods already defined with `*ignored` parameter.